### PR TITLE
Create ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by ioptio**

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- Disk size:
+- Disk Type (HD/SDD):
+- Linux kernel version (uname -a):
+- Compiler version (gcc -version):
+
+### Any extra information that might be useful in the debugging process.
+This includes the relevant contents of `~/.zcash/debug.log`. Raw text or a link to a pastebin type site are preferred.
+Please also include any non-standard things you did during compilation (extra flags, dependency version changes etc.) if applicable.
+
+### Do you have a back up of `~/.zcash directory` and/or take a VM snapshot?
```
Comment (str4d) [inline ISSUE_TEMPLATE.md]: `of the '~/.zcash' directory` (with the correct ticks, as this comment gets rendered as Markdown).

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<!--- Remove text and sections that do not apply -->
+
+This issue tracker is only for technical issues related to zcashd.
+
+General zcash questions and/or support requests and are best directed to the [Zcash Forums](https://forum.z.cash) or [Community Rocket.Chat](https://chat.zcashcommunity.com).
```
Comment (str4d) [inline ISSUE_TEMPLATE.md]: `General Zcash questions`

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+For reporting security vulnerabilities or for sensitive discussions with our security team, please contact [security@z.cash](mailto:security@z.cash). You can use the [GPG key](https://z.cash/gpg-pubkeys/security.asc) (fingerprint: `AF85 0445 546C 18B7 86F9  2C62 88FB 8B86 D8B5 A68C`) to send an encrypted message. The key and fingerprint are duplicated on our [Public Keys page](https://z.cash/support/pubkeys.html).
+
+### Describe the issue
+
```
Comment (str4d) [inline ISSUE_TEMPLATE.md]: Add one or two more newlines here, and maybe a placeholder (e.g. `[ISSUE DESCRIPTION]`) to show that text should be entered here.

File: ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+- OS name + version:
+- CPU:
+- RAM:
+- Disk size:
+- Disk Type (HD/SDD):
+- Linux kernel version (uname -a):
+- Compiler version (gcc -version):
+
+### Any extra information that might be useful in the debugging process.
+This includes the relevant contents of `~/.zcash/debug.log`. Raw text or a link to a pastebin type site are preferred.
```
Comment (str4d) [inline ISSUE_TEMPLATE.md]: Uploading a file inside GitHub is fine too; possibly preferable to a pastebin site, as then we aren't dependent on a secondary service for issue data. But then do we need to specify this? cc/ @nathan-at-least @ebfull @bitcartel @daira for what you'd like to see.

**Comment by ebfull** [comment]
Thank you @ioptio.

ACK

@zkbot r+

**Comment by str4d** [comment]
@zkbot r- since there's a test regression that will cause this to fail.

**Comment by str4d** [comment]
@zkbot r+

**Pull request merged**